extensions/amp-sona/0.1/amp-sona.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
Can you update this value to `2020`? The value should match the year the new file is committed.

Reply:
👍